client/src/Api.js

```diff
@@ -133,6 +133,9 @@ const Api = {
     get(id) {
       return instance.get(`/api/categories/${id}`);
     },
+    getSlug(slug) {
+      return instance.get(`/api/categories/${slug}/resources`);
+    },
```

Comment:
I don't think we need this new Api function... you already previously added a categoryId parameter to the Api.resources.index() api function. We should be able to pass a category id or slug into that function and get the list of resources.

Reply:
Note that, although you added that parameter in Api.js, I don't see an implementation for it in routes/api/resources.js...

Reply:
I removed the function `getSlug()`